pyquil/quilatom.py

```diff
@@ -34,6 +35,13 @@
 
 import numpy as np
 
+from qcs_sdk.quil.instructions import (
+    Qubit as RSQubit,
+    FunctionCallExpression as RSFunctionCallExpression,
+    MemoryReference as RSMemoryReference,
+    Expression as RSExpression,
+)
```

Comment:
What about `import qcs_sdk.quil as quil_rs` or similar? Then there's only one alias and the individual uses are easy to distinguish because they're all prefixed.

Reply:
Sounds good, done!